config/initializers/omniauth.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :facebook, '965297906900288', 'fbd092ba1e3743698abe8c4c6c1fa835'
```

Comment:
These should probably be in your environment rather than your github repo - strangers could nab and use them.

ENV['FACEBOOK_SECRET'] here

then on your dev machine do
export FACEBOOK_SECRET='fdafdafda'
and on heroku set as a config var